### DISCUSSION TITLE
Image not displaying on Gihub Page at end of Markdown Tutorial

### DISCUSSION THREAD

**Discussion by jjJanie** [skills/.github#244] [Q&A] [answered]
At the end of the Communicating using Markdown Tutorial, the "What's Next"  step is to publish GitHub Pages.   I did this but I don't understand why the image I used did not display ?   It displayed okay locally.   Here's a SS of my GitHub Page.
[My GitHub Page] <img width="892" alt="Screenshot 2023-06-01 at 4 53 59 PM" src="https://github.com/skills/.github/assets/134814600/266ab702-5f35-4eb4-af38-f00d84cdc4dd">
And here's the code: 
```
# This is the Biggest Heading, also know as H1
## This is the second biggest heading, known as H2
### The Middle Sized heading, I am H3
#### The inbetweener, H4
##### Second smallest header,  you get the idea the number is 5
###### Smallest header, H6 and six hashtags as you have probably worked out by now
![Fixer boat from Battle Bay game](https://static.wikia.nocookie.net/battlebay_gamepedia_en/images/a/ae/SHIP_REPAIR_6.png/revision/latest?cb=20170209091957)
```
import csv
import pandas as pd
```
- [x] Check Team config of friends and unfriends
- [ ] Proceed to capture at speed comensurate with your team
- [ ] Place Plasma ahead of you in spot useful for fast boats to take cover
```

**Comment by sinsukehlab** [comment, marked as answer]
Why don't you delete the letters beyond the `.png`?

**Reply by jjJanie** [reply]
Thanks  - that fixed it !  I will remember that for next time I copy an image link

**Reply by sinsukehlab** [reply]
I'm glad I could help you.
